Gemfile

```diff
@@ -21,7 +21,7 @@ gem 'hiredis'
 gem 'http_accept_language'
 gem 'httparty'
 gem 'identity-hostdata', github: '18F/identity-hostdata', branch: 'master'
-gem 'identity-telephony', github: '18f/identity-telephony', tag: 'v0.0.17'
+gem 'identity-telephony', github: '18f/identity-telephony', branch: 'margolis-multiple-pinpoint-configs'
```

Comment:
Once the other PR (https://github.com/18F/identity-telephony/pull/25) lands I'll fix this to point at a tag again